tests/core/test_storage_migrations.py

```diff
@@ -8,9 +8,72 @@
 from kinto.core.cache import postgresql as postgresql_cache
 from kinto.core.permission import postgresql as postgresql_permission
 from kinto.core.storage import postgresql as postgresql_storage
+from kinto.core.storage.postgresql.migrator import Migrator
 from kinto.core.testing import skip_if_no_postgresql
 
 
+class MigratorTest(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.migrator = Migrator()
```

Comment:
This looks dangerous. You create an instance of `Migrator` but mutate it within the tests. 
It'd be much better to ...:
```python
    def setUp(self):
        self.migrator = Migrator()
        ...
```

Reply:
Oops, yeah. Good catch. I hate XUnit-style test cases..